book/plugins/general-abilities/video-content-download/video-content-download-general.md

```diff
@@ -0,0 +1,15 @@
+# Video Content Download
+
+`Video Content download` ability uses the provided atom entry video url to download and save the content locally for offline playback.
```

Comment:
How much of this is duplicate from the iOS (or even worse - written twice with different styles)?